ui/selectors/custom-gas.js

```diff
@@ -105,6 +105,10 @@ export function isCustomPriceSafeForCustomNetwork(state) {
     return true;
   }
 
+  if (!estimatedPrice) {
```

Comment:
While not strictly required to fix the bug this PR aims to correct, this ensures that no errors are hit if on a testnet and `state.gas.basicEstimates.average` somehow got set to a falsy value. So this will prevent possible big number errors in the future.